lib/agent0/agent0/interactive_fuzz/fuzz_long_short_maturity_values.py

```diff
@@ -53,69 +58,29 @@ def main(argv: Sequence[str] | None = None):
         logging.info("index=%s\n", index)
         if isinstance(trade, OpenLong):
-            close_long_event = agent.close_long(maturity_time=trade.maturity_time, bonds=trade.bond_amount)
-            # 0.05 would be a 5% fee.
-            flat_fee_percent = interactive_hyperdrive.hyperdrive_interface.pool_config.fees.flat
-
-            # assert with trade values
-            # base out should be equal to bonds in minus the flat fee.
-            actual_base_amount = close_long_event.base_amount
-            # expected_base_amount_from_event = (
-            #     close_long_event.bond_amount - close_long_event.bond_amount * flat_fee_percent
-            # )
-            # assertion(actual_base_amount == expected_base_amount_from_event)
-
-            # assert with event values
-            expected_base_amount_from_trade = trade.bond_amount - trade.bond_amount * flat_fee_percent
-            # assertion(close_long_event.base_amount == expected_base_amount_from_trade)
-
-            # show the difference
-            difference = actual_base_amount.scaled_value - expected_base_amount_from_trade.scaled_value
-            logging.info("close long: actual_base_amount=%s", actual_base_amount.to_decimal())
-            logging.info("close long: difference in wei =%s\n", difference)
-            # assert actual_base_amount == expected_base_amount
-            # assert close_long_event.base_amount == trade.bond_amount - trade.bond_amount * flat_fee_percent
-        if isinstance(trade, OpenShort):
-            close_short_event = agent.close_short(maturity_time=trade.maturity_time, bonds=trade.bond_amount)
-
-            # get the share prices
-            open_share_price = starting_checkpoint.share_price
-            closing_share_price = FixedPoint(scaled_value=maturity_checkpoint.sharePrice)
-
-            # interested accrued in shares = (c1 / c0 + flat_fee) * dy - c1 * dz
-            flat_fee_percent = interactive_hyperdrive.hyperdrive_interface.pool_config.fees.flat
-
-            # get the share amount, c1 * dz part of the equation.
-            share_reserves_delta = trade.bond_amount
-            flat_fee = trade.bond_amount * flat_fee_percent
-            share_reserves_delta_plus_flat_fee = share_reserves_delta + flat_fee
-
-            # get the final interest accrued
-            interest_accrued = (
-                trade.bond_amount * (closing_share_price / open_share_price + flat_fee_percent)
-                - share_reserves_delta_plus_flat_fee
-            )
+            close_event = agent.close_long(maturity_time=trade.maturity_time, bonds=trade.bond_amount)
+        elif isinstance(trade, OpenShort):
+            close_event = agent.close_short(maturity_time=trade.maturity_time, bonds=trade.bond_amount)
+        else:
+            assert False
+
+        if invariant_check_failed(
+            trade, close_event, starting_checkpoint, maturity_checkpoint, random_seed, interactive_hyperdrive, chain
```

Comment:
There's a TODO on line 41:

```python
    # TODO: We will want to log the random seed; so remove this pylint disable once we do
    chain, random_seed, rng, interactive_hyperdrive = setup_fuzz(log_filename)  # pylint: disable=unused-variable
```

Now that you're using the variable you can delete the TODO & pylint: disable line comments.